packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -4,116 +4,155 @@ meta:
 
-Okta Identity Engine requires that an assurance specified in the global session policy and in the authentication policy be satisfied before a user can access an app. This is a change from the traditional model of authentication, which evaluates one policy depending on whether the user signs in to the org or directly through the app. When evaluating whether a user is granted access, Identity Engine inspects the context (user itself, device, network, and risk) that the user brings, first at org level and then at app level. It then determines the authentication methods that are offered based on both Global Session Policies and authentication policies.
+You can create custom forms for your sign-in flows with [profile enrollment policies](#profile-enrollment-policies) to progressively build user profiles by collecting profile data incrementally as end users engage with your app.
 
-A global session policy and an authentication policy control the authentication assurance part of your requirements. Other policies, such as an authenticator enrollment policy, password policy, profile policy, and so on, work together to determine the overall authentication experience.
+### Control which application can access what information from your APIs
 
-* [Global session policy](/docs/reference/api/policy/#global-session-policy): Supplies the context necessary for the user to advance to the next authentication step after they are identified by Okta. Global Session Policies control who can have access, and how a user gains access to Okta, including whether they are challenged for additional factors and how long they are allowed to remain signed in before re-authenticating.
+When you want to restrict access to an API based on the calling application, you can create an [access policy](/docs/reference/api/authorization-servers/#policy-object).
 
-  All orgs have a default global session policy that applies to all users. The policy allows access with a password, IdP, or any factor allowed by the authentication policies. You can change this condition or add higher priority rules to the default policy. You can also create new policies and prioritize them over the default. See [Configure a global session policy and an authentication policy](/docs/guides/configure-signon-policy/main/).
+Access policies are also good when you need scopes in addition to the [reserved scopes](/docs/reference/api/oidc/#scopes) that are created with any Okta authorization server. For example, to improve compatibility for an application, you can return additional profile information for the user by [creating custom scopes with corresponding claims](/docs/guides/customize-authz-server/main/#create-scopes) that tie them to a piece of user information.
```

Comment:
```suggestion
Access policies are also useful when you need scopes in addition to the [reserved scopes](/docs/reference/api/oidc/#scopes) that are created with any Okta authorization server. For example, to improve compatibility for an app, you can return additional profile information for the user by [creating custom scopes with corresponding claims](/docs/guides/customize-authz-server/main/#create-scopes) that tie them to a piece of user information.
```

Reply:
I was going to change this to useful, but I don't know why I hesitated. 😅